topic-operator/scripts/topic_operator_run.sh

```diff
@@ -22,6 +26,14 @@ if [ "$STRIMZI_TLS_ENABLED" = "true" ]; then
     fi
 fi
 
+if [ -n "$KAFKA_HEAP_OPTS" ]; then
+    export JAVA_OPTS="${JAVA_OPTS} ${KAFKA_HEAP_OPTS}"
+fi
+
+if [ -n "$KAFKA_JVM_PERFORMANCE_OPTS" ]; then
+    export JAVA_OPTS="${JAVA_OPTS} ${KAFKA_JVM_PERFORMANCE_OPTS}"
+fi
+
```

Comment:
I do not think we should pull the Kafka env var names here. We should use our own.